edi/tests/test_consumer_mixin.py

```diff
@@ -7,13 +7,15 @@
 
 from odoo_test_helper import FakeModelLoader
 
-from .common import EDIBackendCommonTestCase
+from odoo.tests.common import Form
+
+from .common import EDIBackendCommonComponentRegistryTestCase
 
 
 # This clashes w/ some setup (eg: run tests w/ pytest when edi_storage is installed)
 # If you still want to run `edi` tests w/ pytest when this happens, set this env var.
 @unittest.skipIf(os.getenv("SKIP_EDI_CONSUMER_CASE"), "Consumer test case disabled.")
-class TestConsumerMixinCase(EDIBackendCommonTestCase):
+class TestConsumerMixinCase(EDIBackendCommonComponentRegistryTestCase):
```

Comment:
the component savepoint case is enough as you don't register any new component on the fly